internal/rbd/rbd_attach.go

```diff
@@ -203,17 +208,22 @@ func checkRbdNbdTools() bool {
 		if err != nil {
 			log.ExtendedLogMsg("rbd-nbd: nbd modprobe failed with error %v", err)
 
-			return false
+			return false, ""
 		}
 	}
-	if _, _, err := util.ExecCommand(context.TODO(), rbdTonbd, "--version"); err != nil {
-		log.ExtendedLogMsg("rbd-nbd: running rbd-nbd --version failed with error %v", err)
-
-		return false
+	var rbdNbdVer string
+	stdout, stderr, err := util.ExecCommand(context.TODO(), rbdTonbd, "--version")
+	if err != nil || stderr != "" {
+		log.ExtendedLogMsg("rbd-nbd: running rbd-nbd --version failed with error:%v, stderr:%s", err, stderr)
+		return false, ""
+	} else if stdout != "" {
```

Comment:
any specific reason to have this check? instead of that we split stdout and check its length to make sure it's>3?

Reply:
As also suggested by Niels at `checkRbdNbdTools`, let's also safely ignore a case that doesn't return the version?